### DISCUSSION TITLE
Drag Spectogram

### DISCUSSION THREAD

**Discussion by fatehux** [katspaugh/wavesurfer.js#3633] [Q&A] [answered]
Hello guys,
When i add Spectogram plugin and drag FFT chart but Wavesurfer does not update or drag the spectrogram accordingly, how can i fix that?

![Capture](https://github.com/katspaugh/wavesurfer.js/assets/160303879/af2aae71-9c6c-4b3f-9e10-8a81f93bc99a)

**Comment by katspaugh** [comment]
Not sure I follow. Why do you expect the spectrogram to update or drag?

**Reply by fatehux** [reply]
i mean when select a region in FFT chart  wanna selected same region in spectogram too.
![Capture](https://github.com/katspaugh/wavesurfer.js/assets/160303879/6ee386c6-1cc6-43ac-9da9-6540d444d7c1)

**Reply by katspaugh** [reply, marked as answer]
Ah, I see. It looks like the z-index of the spectrogram is higher than a region’s z-index.

**Reply by fatehux** [reply]
Many thanks.
